Podcast/EpisodeCard.swift

```diff
@@ -25,6 +25,7 @@ class EpisodeCard: Card {
     var isRecommended = false
     var isPlaying = false
     
+    //initializer with all attributes
     init(episodeID: Int, episodeTitle: String = "", dateCreated: Date = Date(), descriptionText: String = "Not avaliable", smallArtworkImageURL: URL, episodeLength: Double = 0.0, numberOfRecommendations: Int = 0, tags: [Tag] = [], seriesTitle: String = "", isBookmarked: Bool = false, isRecommended: Bool = false) {
```

Comment:
you don't need to give default values for variables such as episodeTitle, or any variables in general, they're not optional so they will have to be set to something by whatever is calling the initializer